e2e-tests/cypress.json

```diff
@@ -5,6 +5,9 @@
   "viewportWidth": 1024,
   "viewportHeight":1600,
   "env": {
-    "demoDelay": 0
+    "demoDelay": 0,
+    "RMQport": 4004,
```

Comment:
these RMQport etc. fields will presumably want to match all the "HORIZON_HAS_PUBLISHER_xxx" fields in the docker-compose - + it looks like you're considering queuename configurable which makes sense and we should add that to our cypress config

Reply:
I am happy with similar or different names. I took these from your PR.

Reply:
(yeah my PR came in before i saw what you'd called things in docker-compose ;) all good, we'll pick this up as we integrate the glue code)